### PR TITLE
fix(one-app-bundler): maxEntrypointSize and increase default to 250e3

### PR DESCRIPTION
**Pull request by infoxicator**
Added missing `maxEntrypointSize` to the webpack client configuration.

Changed the default value for both `maxAssetSize` and `maxEntrypointSize` to 250e3 instead of 200e3.

We previously discussed about increasing the default performance budget, and seems like the 250e3 is a sane default also used by `webpack` in their official docs 
https://webpack.js.org/configuration/performance/#performancemaxentrypointsize

### CODE REVIEW

File: packages/one-app-bundler/webpack/module/webpack.client.js [hunk 1]
```diff
@@ -58,7 +58,8 @@ module.exports = (babelEnv) => {
       },
       node: { module: 'empty', net: 'empty', fs: 'empty' },
       performance: {
-        maxAssetSize: configOptions.performanceBudget || 200e3,
+        maxAssetSize: configOptions.performanceBudget || 250e3,
+        maxEntrypointSize: configOptions.performanceBudget || 250e3,
```
Comment (JamesSingleton) [inline packages/one-app-bundler/webpack/module/webpack.client.js:62]: Would we ever want these to be different?

Comment (Francois-Esquire) [reply packages/one-app-bundler/webpack/module/webpack.client.js:62]: For the defaults? What were you thinking?

Comment (JamesSingleton) [reply packages/one-app-bundler/webpack/module/webpack.client.js:62]: @Francois-Esquire meaning should I be able to set `maxAssetSize` and `maxEntrypointSize` independently?

Comment (10xLaCroixDrinker) [reply packages/one-app-bundler/webpack/module/webpack.client.js:62]: That can be done via custom webpack config

**Pull request merged**